test/tests/multiapps/picard_multilevel/2level_picard/sub_level2.i

```diff
@@ -0,0 +1,78 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 0
+  xmax = 1
+  ymin = 0
+  ymax = 1
+  nx = 10
+  ny = 10
+[]
+
+[Variables]
+  [w]
+  []
+[]
+
+[AuxVariables]
+  [v]
+  []
+[]
+
+[Kernels]
+  [time_derivative]
+    type = TimeDerivative
+    variable = w
+  []
+  [diffusion]
+    type = Diffusion
+    variable = w
+  []
+  [source]
+    type = CoupledForce
+    variable = w
+    v = v
+  []
+[]
+
+[BCs]
```

Comment:
Could you please replace this BC with:

```
[BCs]
  [dirichlet0]
    type = DirichletBC
    variable = w
    boundary = '3'
    value = 0
  []
  [dirichlet]
    type = DirichletBC
    variable = w
    boundary = '1'
    value = 100
  []
[]
```

That way, the converged solution for w should be the same as the one for u. It makes for a nice verification. I did check that this is indeed the case if the Picard tolerances are tightened.

Reply:
Yes